addon/classes/Table.js

```diff
@@ -139,42 +139,46 @@ export default class Table extends Ember.Object.extend({
    * Replace all the row's content with content of the argument. If argument is an empty array rows will be cleared.
    * @method setRows
    * @param  {Array} rows
+   * @param  {Object} options
    * @return {Array} rows
    */
-  setRows(rows = []) {
-    return this.rows.setObjects(Table.createRows(rows));
+  setRows(rows = [], options = {}) {
+    return this.rows.setObjects(Table.createRows(rows, options));
   }
 
   /**
    * Push the object onto the end of the row array if it is not already present.
    * @method addRow
    * @param  {Object} row
+   * @param  {Object} options
    */
-  addRow(row) {
+  addRow(row, options = {}) {
     if (row instanceof Row) {
       this.rows.addObject(row);
     } else if (isNone(this.rows.findBy('content', row))) {
-      this.pushRow(row);
+      this.pushRow(row, options);
     }
   }
 
   /**
    * Push the objects onto the end of the row array if it is not already present.
    * @method addRows
    * @param  {Array} rows
+   * @param  {Object} options
    */
-  addRows(rows = []) {
-    rows.forEach(r => this.addRow(r));
+  addRows(rows = [], options = {}) {
```

Comment:
@curtisgr why's that? `options` passed to `addRows` will be applied to all row objects.